cmd/sfncli/runner.go

```diff
@@ -78,6 +80,10 @@ func (t *TaskRunner) Process(ctx context.Context, args []string, input string) e
 	if executionName != nil {
 		t.execCmd.Env = append(os.Environ(), "_EXECUTION_NAME="+*executionName)
 	}
+	if t.workDirectory != "" {
+		t.execCmd.Env = append(t.execCmd.Env, "WORK_DIR="+t.workDirectory)
```

Comment:
I considered appending the `_EXECUTION_NAME` to the `workDirectory` here but that seems like magic that is not really required.